test/codegen-generated-classes-test/src/main/resources/codegen-resources/endpointproviders/endpoint-rule-set.json

```diff
@@ -27,12 +27,12 @@
             "documentation": "Override the endpoint used to send this request",
             "type": "String"
         },
-        "awsAccountId": {
+        "AccountId": {
```

Comment:
Why are we changing this ?  Did the specifications change ?  I remember discussing about this in design meeting that the spec said it is "awsAccountId" , did it get changed ?